src/backend/distributed/deparser/citus_ruleutils.c

```diff
@@ -422,7 +422,9 @@ pg_get_tableschemadef_string(Oid tableRelationId, bool includeSequenceDefaults)
 		/* deparse check constraint string */
 		char *checkString = deparse_expression(checkNode, checkContext, false, false);
 
+		appendStringInfoString(&buffer, "(");
```

Comment:
nit; When appending a single char `appendStringInfoChar` is faster than `appendStringInfoString`.

See comments in PostgreSQL code:
https://github.com/postgres/postgres/blob/8255c7a5eeba8f1a38b7a431c04909bde4f5e67d/src/include/lib/stringinfo.h#L108-L120